### ISSUE TITLE
How to define the prefix of the deployments?

### ISSUE THREAD

**Issue by meir-doit-intl** [alenkacz/cert-manager-verifier#1]
The prefix of my deployments is `cert-manager-release-...`, so the current hardcoded solution doesn't work. Is there a flag/argument that can be used to change the prefix?

```
$ cm-verifier --namespace cert-manager-v2
Waiting for deployments in namespace cert-manager-v2:
Deployment cert-manager not found. Required?: true
Deployment cert-manager-cainjector not found. Required?: false
Deployment cert-manager-webhook not found. Required?: false
Error: FAILED! Not all deployments are ready.
```

Thanks

**Comment by alenkacz** [comment]
Not right now, I'll add this. Thanks for the issue and sorry it took me time to reply. I'll try to get this released ASAP

**Comment by alenkacz** [comment]
This is fixed in 0.2.0
